orderbook/openapi.yml

```diff
@@ -118,22 +118,55 @@ paths:
         404:
           description: Order was not found
     delete:
-      summary: Deletes order from order service,
+      summary: Deletes order from order service
       description: |
         The successful deletion might not prevent solvers from settling the order
         Authentication must be provided by signing the following message:
-        signTypedData({operation: "delete", orderUid: "0x.."})
       parameters:
         - in: path
           name: UID
           schema:
             $ref: "#/components/schemas/UID"
           required: true
+      requestBody:
+        description: The order to cancel.
```

Comment:
The body is the EIP712 signature of `struct OrderCancellation { orderUid: bytes }` from the order's owner.


nit: no strong feelings, but maybe having the request body be an object with field signature instead of directly being a string might make it easier to add other parameters later if we decided to do so.

Reply:
I'm pretty sure the order body is the Signature struct... as declared a few lines below here... Is that not what you mean?

Reply:
Or do you mean that the `#/components/schemas/Signature` is not well defined as is?

Reply:
What's the type of Signature? It's not defined as `type: object` so I'd assume it's a raw `string`. This means a valid body would be

```
"0xff...fff"
```

and I think it would be nicer to have the body be:

```
{ "signature": "0xff...fff" }
```

Reply:
Gotcha!

Reply:
I have updated this according to your input, but now have the same string in both `requestBody.description` and `OrderCancellation.description`. I would think that we might change the request body description to just say "Signed OrderCancellation" if we want to avoid redundancy.

Reply:
Sounds good to me. If you remove the commented code and move this PR from Draft I think this is good to go.

Reply:
Whoops, I already started implementation. Will have to separate these.